incubator/kafka/values.yaml

```diff
@@ -134,6 +134,52 @@ persistence:
   ##
   # storageClass:
 
+## Metrics
+##
+metrics:
+
+  jmx:
+    enabled: false
+
+    # The image to use for the metrics collector
+    image: solsson/kafka-prometheus-jmx-exporter@sha256
+
+    # The image tag to use for the metrics collector
+    imageTag: a23062396cd5af1acdf76512632c20ea6be76885dfc20cd9ff40fb23846557e8
+
+    # The port to expose JMX metrics on
+    port: 5555
+
+    # Rules to apply to the Kafka JMX Exporter
+    kafkaConfig:
```

Comment:
do you think we could drop this key and tab everything over?

Reply:
I don't think so due to the way the key is used; ie, the blob is injected as `{{ toYaml .Values.metrics.jmx.kafkaConfig | indent 4 }}`. If we were to drop the key and unindent, we'd end up including all of the other `.metrics.jmx` values in the `jmx-kafka-prometheus.yml` configuration. I'm not sure if that will cause the jmx exporter to error on startup, but it seems unclean either way.

Reply:
ah yeah, of course 👍